src/DynamoUtilities/Guid.cs

```diff
@@ -118,5 +120,31 @@ private static void SwapBytes(byte[] guid, int left, int right)
             guid[left] = guid[right];
             guid[right] = temp;
         }
+
+        /// <summary>
+        /// Performs an update to all Guids inside the json string before deserialization.
+        /// Targets specifically Guids without the '-' hyphen, which are all the workspace elements.
+        /// Replacing all occurrences of each individual Guid guarantees that the relationships between the elements are retained.
+        /// </summary>
+        /// <param name="jsonData">Json representation of workspace.</param>
+        /// <returns>String representation of workspace after all elements' Guids replaced.</returns>
+        internal static string UpdateWorkspaceGUIDs(string jsonData)
+        {
+            string pattern = @"([a-z0-9]{32})";
```

Comment:
is it possible that this will match a node name etc.

For example, let's say I create a node a guid in the name - this code will modify it- correct? Is that okay?

Reply:
Unless the node name is 32 digit without space, otherwise not affected